jobs/rocoto/awips.sh

```diff
@@ -1,4 +1,9 @@
-#!/bin/bash -x
+#! /usr/bin/env bash
```

Comment:
I also don't understand this job.
It doesn't have a typical j-job?
How is this being called in ops?

Reply:
The jjobs are coded in `AWIPS20SH` and `AWIPSG2SH`, which are set in [config.awips](https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/config.awips). Those are then called by this script.

Yeah, this arrangement doesn't make sense to me either. Still really want to get rid of all these `jobs/rocoto/` scripts anyway and call j-jobs directly.

Reply:
I see whats going with `awips.sh`.
This is being treated like `arch.sh` or any other without a j-job.
The j-job does not follow the convention of the other global-workflow j-jobs.
The configs are sourced in this rocoto wrapper job and probably not in the j-job.